client/db_test.go

```diff
@@ -288,3 +290,46 @@ func TestDebugName(t *testing.T) {
 		return nil
 	})
 }
+
+func TestCommonMethods(t *testing.T) {
+	types := []reflect.Type{
+		reflect.TypeOf(&client.DB{}),
+		reflect.TypeOf(&client.Tx{}),
+		reflect.TypeOf(&client.Batch{}),
+		reflect.TypeOf(client.DB{}.B),
+	}
+	blacklist := map[string]bool{
+		"AdminMerge":          true,
+		"AdminSplit":          true,
+		"Commit":              true,
+		"DebugName":           true,
+		"InternalAddCall":     true,
+		"InternalKV":          true,
+		"InternalSetPriority": true,
+		"Run":                  true,
```

Comment:
Not sure why `gofmt` is producing this funny indentation. But it is doing so repeatedly.